rules/windows/builtin/system/microsoft_windows_kernel_general/win_system_susp_critical_hive_location_access_bits_cleared.yml

```
@@ -0,0 +1,32 @@
+title: Critical Hive In Suspicious Location Access Bits Cleared
+id: 39f919f3-980b-4e6f-a975-8af7e507ef2b
+related:
+    - id: 839dd1e8-eda8-4834-8145-01beeee33acd
+      type: obsoletes
+status: test
+description: |
+    Detects events from the Kernel-General ETW indicating that the access bits of a hive with a system like hive name located in the temp directory have been reset.
+    This occurs when an application tries to access a hive and the hive has not be recognized since the last 7 days (by default).
+    Registry hive dumping utilities such as QuarksPwDump were seen emitting this behavior.
+references:
+    - https://github.com/nasbench/Misc-Research/blob/b20da2336de0f342d31ef4794959d28c8d3ba5ba/ETW/Microsoft-Windows-Kernel-General.md
+author: Florian Roth (Nextron Systems)
+date: 2017/05/15
+modified: 2024/01/18
+tags:
+    - attack.credential_access
+    - attack.t1003.002
+logsource:
+    product: windows
+    service: system
+detection:
+    selection:
+        EventID: 16
+        Provider_Name: Microsoft-Windows-Kernel-General
+        HiveName|contains:
+            - 'Temp\SAM'
```

```suggestion
            - '\Temp\SAM'
```
Is the missing `\` intentional?